_rules/SC2-4-2-page-has-title.md

```diff
@@ -34,7 +34,7 @@ The page contains at least one `title` element.
 
 ### Expectation 2
 
-The first `title` element contains [non-empty text](#non-empty).
+The first `title` element has [text nodes](https://www.w3.org/TR/dom/#text) that do not only have [whitespace](#whitespace).
```

Comment:
I think talking about "text nodes" is confusing. As best I can tell, a `title` element can not have more than one text node, similar to `script` or `style`.

Reply:
Not quite: https://github.com/act-rules/act-rules.github.io/pull/380#discussion_r275313664

Reply:
Agreed in https://github.com/act-rules/act-rules.github.io/pull/380#discussion_r275313664 to leave it as it is.